app/javascript/pages/listings/listing-detail.tsx

```diff
@@ -298,34 +300,71 @@ const ListingDetail = () => {
+                {renderInlineWithInnerHTML(stripMostTags(listing.Required_Documents))}
               </div>
             </div>
-          )}
-          {listing.Legal_Disclaimers && (
-            <div className="listing-detail-panel">
-              <div className="info-card">
-                <h3 className="text-serif-lg">Important Program Rules</h3>
-                <p className="text-sm text-gray-700 m-0 p-0">
-                  <MarkdownSection>{listing.Legal_Disclaimers}</MarkdownSection>
-                </p>
+            <div className="info-card bg-gray-100 border-0">
+              <h3 className="text-serif-lg">{t("listings.important_program_rules")}</h3>
+              <div className="text-sm">
+                {renderInlineWithInnerHTML(stripMostTags(listing.Legal_Disclaimers))}
               </div>
             </div>
-          )}
-          <div className="listing-detail-panel">
-            <div className="info-card">
-              <h3 className="text-serif-lg">
-                Monitored by the Mayor's Office of Housing and Community Development
-              </h3>
+            {listing.CC_and_R_URL && (
+              <div className="info-card bg-gray-100 border-0">
+                <h3 className="text-serif-lg">{t("listings.cc&r")}</h3>
+                <div className="text-sm">
+                  {renderInlineWithInnerHTML(stripMostTags(listing.CC_and_R_URL))}
+                </div>
+              </div>
+            )}
+            {/*
+            {listing.isSale && (
+            <div className="info-card bg-gray-100 border-0">
+              <h3 className="text-serif-lg">For the Buyer's Realtor</h3>
+              {listing.Allows_Realtor_Commission ? (
+                display realtor_commission_header
+                realtorComissionMessage
+                {listing.Realtor_Commission_Info && realtor_commission_how_to}
+              ) : display realtor_commission_not_eligible message}
             </div>
+            )}
+            */}
+            {listing.Repricing_Mechanism && (
+              <div className="info-card bg-gray-100 border-0">
+                <h3 className="text-serif-lg">{t("listings.re_pricing")}</h3>
+                <div className="text-sm">
+                  {renderInlineWithInnerHTML(stripMostTags(listing.Repricing_Mechanism))}
+                </div>
+              </div>
+            )}
           </div>
         </ListingDetailItem>
 
         {getSidebar()}
+        <div className="listing-detail-panel">
+          <div className="info-card flex">
+            {/* TODO: do we have a class for serifs but smaller we can enable? */}
+            <p className="text-serif-lg">
+              Monitored by the Mayor's Office of Housing & Community Development
+            </p>
+            <img
+              alt="Equal Housing Opportunity Logo"
+              src="/assets/logo-equal-784f0277ac265fa4f14d489618f11b326fcdf72d4ce37b4d83607b3b0c37c6b1.png"
```

Comment:
[an example](https://github.com/SFDigitalServices/sf-dahlia-web/blob/054db9e7fb6e3c3a14ec5492def8d189ead6825e/app/javascript/layouts/Layout.tsx#L164-L164) of how to get an asset